thriftpy2/parser/parser.py

```diff
@@ -404,6 +409,16 @@ def set_info(self, info):
         return self.index + 1
 
 
+class CurrentFieldSeqImplicitId(dict):
```

Comment:
I think this is an anti-pattern in OOP. It's not a dict and doesn't behave like one, so it shouldn't be a subclass of dict. You can just let it inherit from object directly and use a dict as an instance variable.

Reply:
Sure thing, can change. For context this was modeled off the `CurrentIncompleteType` class that already existed in this module, and does inherit from `dict`:

https://github.com/Thriftpy/thriftpy2/blob/56c6df707e282d6f1bf795667dda27c12af36b56/thriftpy2/parser/parser.py#L403-L409